docker/rpm/Dockerfile

```diff
@@ -58,6 +58,19 @@ COPY entrypoint.sh /usr/bin/entrypoint.sh
 RUN chmod +x /usr/bin/entrypoint.sh
 ENTRYPOINT [ "/usr/bin/entrypoint.sh" ]
 
-FROM builder as uploader
-RUN yum install -y gnupg rpm-sign && \
-    pip install boto awscli
+
+# NOTE: uploading with rpm-s3 requires python2
```

Comment:
Is there anything critical that needs conversion? We use https://github.com/efiop/rpm-s3

Reply:
Just wondering if it is easy to 2to3 it.

Reply:
We could port rpm-s3 to python3.

Running `2to3` on `bin/rpm-s3` only changes ~11 lines, but there's also vendored dependencies which are also python2 as well as some deprecated modules which would need to be removed